src/dbnode/storage/types.go

```diff
@@ -1302,12 +1303,13 @@ type NewBackgroundProcessFn func(Database, Options) (BackgroundProcess, error)
 // AggregateTilesOptions is the options for large tile aggregation.
 type AggregateTilesOptions struct {
 	// Start and End specify the aggregation window.
-	Start, End          time.Time
+	Start, End time.Time
 	// Step is the downsampling step.
-	Step                time.Duration
+	Step time.Duration
 	// HandleCounterResets is temporarily used to force counter reset handling logics on the processed series.
 	// TODO: remove once we have metrics type stored in the metadata.
 	HandleCounterResets bool
+	MetricsScope        tally.Scope
```

Comment:
nit: would this make more sense as an `instrument.Options`?

Reply:
The good thing about the `tally.Scope` is that I can set tags. With `instrument.Options` I didn't find how to do it?

Reply:
```golang
iOpts := instrument.NewOptions()
// To get scope
scope := iOpts.MetricsScope()
// To set scope on opts
iOpts = iOpts.SetMetricsScope(scope.With("tags...."))
```

Reply:
It just seems more conventional to put instrument options into options structs rather than the raw scopes